src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Assessment.kt

```diff
@@ -33,6 +33,9 @@ data class Assessment(
 
   @NotNull
   val lastUpdatedTimestamp: LocalDateTime = LocalDateTime.now(),
+
+  @NotNull
+  val policyVersion: String = "???",
```

Comment:
this should always be set when creating an assessment. 
Shame I had to provide a default value but didn't want kotlin to think it could be null.. probably not a massive deal but not sure if there is a way around this?

Reply:
Could we have a policy version enum like CVL API?

Reply:
I see you do have PolicyV1 class, can't you just use PolicyV1.code?

Reply:
yes! that'd be good